### PR TITLE
PE-37 | Fixed Side Menu and added styling to the UI

### PR DESCRIPTION
**Pull request by ayeejane**
What I Did:

- Fixed the side menu. It is now accessible for all of the screens: `Home`, `Profile`, `Create New Post`, and `Settings`.
- I also added some styling to the side menu as well as change the color of the **appBar** of each screens. 

How to Test: 

- Click on the `Side Menu` from the `Home` screen and see if it expands. 
- Click on each screen and see if it redirects you to that specific screen you selected.
- On each screen, the `Side Menu` should be accessible.

### CODE REVIEW

File: lib/components/side_menu.dart [hunk 1]
```diff
@@ -10,37 +11,51 @@ class SideMenu extends StatelessWidget {
...
-            sideMenuListTile(context, 'Settings', SettingsScreen()),
-          ],
+    return Drawer(
+        child: Column(children: <Widget>[
+      UserAccountsDrawerHeader(
+        decoration: BoxDecoration(
+          color: Colors.blueGrey[800],
         ),
+        accountName: Text('Tony N.'),
+        accountEmail: Text('tonynguyen@gmail.com'),
```
Comment (dominikyz) [inline lib/components/side_menu.dart:21]: this looks great, just question: do we have ability to pull username from our session yet?

File: lib/components/side_menu.dart [hunk 2]
```diff
@@ -10,37 +11,51 @@ class SideMenu extends StatelessWidget {
...
+        Icons.settings,
+      ),
+      Divider(color: Colors.grey.shade400),
+    ]));
   }
 }
 
 ListTile sideMenuListTile(
-    BuildContext context, String string, Widget newScreen) {
+    BuildContext context, String string, Widget newScreen, IconData leading) {
```
Comment (dominikyz) [inline lib/components/side_menu.dart:52]: Just checking whether should it be  IconData or  iconData

**Review by dominikyz — APPROVED** [review]
Code looks great! Everything works as intended, I can access side menu from any page now. I've put couple comments in review, but they are super minor, so I'll approve it now . 
Great Job ! :)

File: lib/components/side_menu.dart (hunk 1 above)
Comment (ayeejane) [reply lib/components/side_menu.dart:21]: not yet, login has to be merged in first

**Review by MaxLian11 — COMMENTED** [review]
Seems like everything works fine!
Is there any chance to position the side menu button and the rest of the appBar of the home screen at the same height as it is in other screens? Also, are all other pages going to inherit the style? Like different settings pages? 
Otherwise, looks great.

**Review by MaxLian11 — APPROVED** [review]

**Review by tonynguyen98 — APPROVED** [review]
Was able to run this on my system and it work great! Good job!!

**Comment by ayeejane** [comment]
> Seems like everything works fine!
> Is there any chance to position the side menu button and the rest of the appBar of the home screen at the same height as it is in other screens? Also, are all other pages going to inherit the style? Like different settings pages?
> Otherwise, looks great.

I have to look into it more but it's really just an UI issue.

**Pull request merged**